packages/components/button/src/variants/tinted.ts

```diff
@@ -0,0 +1,88 @@
+const tw = <T>(a: T): T => a
```

Comment:
This function definition is repeated several times. Can we do something regarding that ? I think that is not needed